src/Widget/FieldPart/Hint.php

```diff
@@ -147,7 +149,7 @@ private function getAttribute(): string
     private function getFormModel(): FormModelInterface
     {
         if ($this->formModel === null) {
-            throw new InvalidArgumentException('Form model is not set.');
+            throw new FormModelNotSetException('Failed to create widget because form model is not set.');
```

Comment:
Are these the same messages as in the class itself? How about removing constructor argument and making these part of the exception class?

Reply:
done.

Reply:
done.